smarts/core/utils/traffic_history_serivce.py

```diff
@@ -163,6 +166,18 @@ def fetch_history_at_timestep(self, timestep: str):
 
     @staticmethod
     def fetch_agent_missions(history_file_path):
+        history_mission_filepath = os.path.join(
+            os.path.dirname(utils.__file__), "temp_vehicle_mission.pkl"
+        )
```

Comment:
Perhaps this get name operation could be made into a function in __init__.py of utils
```suggestion
        )
        def get_path_name(file_name):
            return os.path.join(
               os.path.dirname(utils.__file__), file_name
            )
```

then here
`history_mission_filepath = get_path_name("temp_vehicle_mission.pkl")`

Reply:
Thanks,